services/api-server/src/simcore_service_api_server/api/routes/files.py

```diff
@@ -295,6 +295,7 @@ async def complete_multipart_upload(
 @cancel_on_disconnect
 async def abort_multipart_upload(
     request: Request,
+    file_id: UUID,
```

Comment:
why have this parameter here since clearly it is not required by the function?

Reply:
Should we not change the validation at this point? @sanderegg any idea what this will be required even if not used?

Reply:
Good point. I welcome any suggestions regarding this. The point is that the the abortion is associated with the file as a resource. So from a REST/HTTP point of view I think the `file_id` should be a path parameter. The point is that `abort_upload_link` should match the `file_id`. I was looking for a good way to verify that this is the case but couldn't find one (any suggestion @sanderegg?)

Reply:
Ah sorry. I think I found the solution now. I will commit my changes

Reply:
well from the point of view of the client of api-server I guess it makes sense.
maybe a few options are available:
- using file_id to reconstruct the abort link (not sure if it's possible)
- instead of having the abort call such as /{file_id}:abort, have it similar to storage /futures/{future_id}:abort

I see now that you have to pass the abort link in the body right? this is a bit weird from the client code point of view isn't it?
Then maybe that would tend to suggest option 2 would be the better one

Reply:
So I think I finally got this to work in the following way:
For the `complete` and `abort` endpoints one has the pass the `file_id` as a path parameter and a `client_file` (which just holds the filename. From that the link to storage is constructed and and storage is called. Does this sound OK to both of you?